src/main/java/net/atos/zac/documenten/InboxDocumentenService.java

```diff
@@ -45,20 +44,27 @@ public class InboxDocumentenService {
     @PersistenceContext(unitName = "ZaakafhandelcomponentPU")
     private EntityManager entityManager;
 
-    @Inject
     private ZrcClientService zrcClientService;
 
-    @Inject
     private DrcClientService drcClientService;
 
+    @Inject
+    public InboxDocumentenService(
```

Comment:
shouldn't you also inject the entityManager here?

Reply:
yes, you should. and you're now missing the no-arg constructor?